doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -7,22 +7,173 @@ IronSide Secure Element
+* PSA Internal Trusted Storage service
 
-Distribution
-************
+.. _ug_nrf54h20_ironside_se_programming:
 
-The |ISE| is provided as a precompiled binary, which is part of the nRF54H20 SoC bundle and is provided independently from the |NCS| release cycle.
-For more information, see :ref:`abi_compatibility`.
+Programming |ISE| on the nRF54H20 SoC
+*************************************
+
+|ISE| is included in the nRF54H20 SoC binaries.
+The nRF54H20 SoC binaries are released independently from the |NCS| release cycle and are bundled in a ZIP archive that contains the following components:
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Component
+     - File
+     - Description
+   * - IronSide SE update firmware
+     - :file:`ironside_se_update.hex`
+     - The main |ISE| firmware
+   * - IronSide SE Recovery update firmware
+     - :file:`ironside_se_recovery_update.hex`
+     - The recovery firmware
+   * - Update application
+     - :file:`update_application.hex`
+     - The application firmware used to trigger the update process
+
+The bundle ZIP file follows the naming convention :file:`<soc>_soc_binaries_v<version>.zip`.
+
+For more information on the nRF54H20 SoC binaries, see :ref:`nRF54H20 SoC binaries<abi_compatibility>`.
+For instructions on how to program the nRF54H20 SoC binaries, see :ref:`ug_nrf54h20_SoC_binaries`.
+
+By default, the nRF54H20 SoC uses the following memory and access configurations:
+
+* MRAMC configuration: MRAM operates in Direct Write mode.
+* MPC configuration: All memory not reserved by Nordic firmware is accessible with read, write, and execute (RWX) permissions by any domain.
+* TAMPC configuration: The access port (AP) for the application core is enabled and available, allowing direct programming of all the memory not reserved by Nordic firmware in the default configuration.
+
+.. note::
+   Access to external memory (EXMIF) requires a non-default configuration of the GPIO.CTRLSEL register.
+
+Global domain memory can be protected from write operations by configuring UICR registers.
+To remove these protections and disable all other protection mechanisms enforced through UICR settings, perform an ``ERASEALL`` operation.
+
+.. _ug_nrf54h20_ironside_se_update:
+
+Updating |ISE|
+**************
+
+|NCS| supports two methods for updating the |ISE| firmware on the nRF54H20 SoC:
+
+* Using the ``west`` command.
+  You can use the ``west`` command provided by the |NCS| to install the firmware update.
+  For step-by-step instructions, see :ref:`ug_nrf54h20_ironside_se_update_west`.
+
+* Updating the SoC binaries manually.
+  Alternatively, you can perform the update by manually executing the same steps carried out by the ``west`` command.
+  For step-by-step instructions, see :ref:`ug_nrf54h20_ironside_se_update_manual`.
+
+.. caution::
+   You cannot update the nRF54H20 SoC binaries from a SUIT-based (up to 0.9.6) to an IronSide-SE-based (2x.x.x) version.
```

Comment:
```suggestion
   You cannot update the nRF54H20 SoC binaries from a SUIT-based (up to 0.9.6) to an |ISE|-based (20.0.0 and onwards) version.
```

30.0.0 is a valid IronSide version.

As for the `|ISE|` change I'm not sure if it renders identically, just seemed odd to not use the abbreviation.